### PR TITLE
Add readme to amip

### LINKED ISSUE #652: add checklist of AMIP requirements to docs
Before our release, we should make sure that our docs contain everything we want them to, including a checklist of AMIP requirements

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Closes #652 

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by szy21** [comment]
@LenkaNovak @Julians42 I addressed the comments for the previous PR, added some more description to edmf, and moved it to experiments/AMIP/README. For some reason the old PR was not updated so I opened a new one. Would you like to look at it again?

File: experiments/AMIP/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Atmosphere
+
+Dynamical core:
+- Equation: non-hydrostatic and fully compressible
+- Prognostic variables: Density, velocity components, total energy, total specific humidity
+- Spatial discretization: Spectral element in the horizontal, finite difference in the vertical, cubed sphere
+- Time stepping: Implicit-explicit additive Runge–Kutta method
+
+Radiation: A scheme based on RRTM for General circulation model applications—Parallel (RRTMGP) (Pincus et al. 2019)
```
Comment (LenkaNovak) [inline experiments/AMIP/README.md:11]: Maybe call this a `Parameterizations` subsection?

File: experiments/AMIP/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Convection and turbulence: Diagnostic Eddy-diffusivity Mass-Flux (EDMF) scheme with prognostic turbulent kinetic energy. EDMF is a unified parameterization for turbulence and convection. The grid is decomposed into convective updrafts and the turbulent enrionment. Updraft properties are calculated from mass, momentum and energy conservation of the updraft. Mass and momentum exchange between the updrafts and the environment, and
```
Comment (LenkaNovak) [inline experiments/AMIP/README.md:13]: enrionment -> environment

File: experiments/AMIP/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Thermodynamics: 0-layer model, with prognostic ice surface temperature and fixed ice thickness
+
+### Coupling
+
+Sequential coupling. Fluxes over a heterogeneous surface are calculated using the averaged surface properties.
+
+## References
+
+[Manabe 1969](https://journals.ametsoc.org/view/journals/mwre/97/11/1520-0493_1969_097_0739_catoc_2_3_co_2.xml): Climate and ocean circulation. I. The atmospheric circulation and the hydrology of the earth's surface.
+
```
Comment (LenkaNovak) [inline experiments/AMIP/README.md:49]: Can we add references to the EDMF?

File: experiments/AMIP/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Prognostic variables: Density, velocity components, total energy, total specific humidity
+- Spatial discretization: Spectral element in the horizontal, finite difference in the vertical, cubed sphere
+- Time stepping: Implicit-explicit additive Runge–Kutta method
+
+Radiation: A scheme based on RRTM for General circulation model applications—Parallel (RRTMGP) (Pincus et al. 2019)
+
+Convection and turbulence: Diagnostic Eddy-diffusivity Mass-Flux (EDMF) scheme with prognostic turbulent kinetic energy. EDMF is a unified parameterization for turbulence and convection. The grid is decomposed into convective updrafts and the turbulent enrionment. Updraft properties are calculated from mass, momentum and energy conservation of the updraft. Mass and momentum exchange between the updrafts and the environment, and
+turbulent mixing in the environment, are represented with physical closures. Currently, only one updraft is used.
+
+Microphysics: 0-moment scheme, where cloud condensate is removed with a constant timescale
```
Comment (LenkaNovak) [inline experiments/AMIP/README.md:16]: "removed in-situ"?

**Review by LenkaNovak — COMMENTED** [review]
Thanks, @szy21! Just a couple of comments.

**Review by juliasloan25 — COMMENTED** [review]
This looks good to me! I think it's ready to merge after you address Lenka's comments

File: experiments/AMIP/README.md (hunk 3 above)
Comment (szy21) [reply experiments/AMIP/README.md:54]: I added some papers on EDMF. None of them are exactly the same as what we have now, but they are the closest reference before we have a new paper on EDMF.

**Review by LenkaNovak — APPROVED** [review]
LGTM, thank you @szy21

**Pull request merged**